packages/dds/tree/src/test/simple-tree/treeApi.spec.ts

```diff
@@ -95,14 +96,91 @@ describe("treeApi", () => {
 		const root = getView(config).root;
 		const child = root.x;
 		const newChild = new Point({});
-		assert.equal(nodeApi.status(root), TreeStatus.InDocument);
-		assert.equal(nodeApi.status(child), TreeStatus.InDocument);
+		assert.equal(Tree.status(root), TreeStatus.InDocument);
+		assert.equal(Tree.status(child), TreeStatus.InDocument);
 		// TODO: This API layer should have an Unhydrated status:
 		// assert.equal(nodeApi.status(newChild), TreeStatus.Unhydrated);
 		root.x = newChild;
-		assert.equal(nodeApi.status(root), TreeStatus.InDocument);
-		assert.equal(nodeApi.status(child), TreeStatus.Removed);
-		assert.equal(nodeApi.status(newChild), TreeStatus.InDocument);
+		assert.equal(Tree.status(root), TreeStatus.InDocument);
+		assert.equal(Tree.status(child), TreeStatus.Removed);
+		assert.equal(Tree.status(newChild), TreeStatus.InDocument);
 		// TODO: test Deleted status.
 	});
+
+	describe("on", () => {
```

Comment:
As mentioned in the description, these are not new tests, they are copy pasted from node.spec.ts to resolve a TODO there.